### PR TITLE
update rails version up to 4.0.3, add trello gem, add trello credentials to env

### PR DESCRIPTION
**Pull request by dmitrytrager**

### CODE REVIEW

**Comment by IldusSadykov** [comment]
:+1:

File: Gemfile [hunk 1]
```diff
@@ -2,13 +2,15 @@ source 'https://rubygems.org'
...
 
-gem 'rails', '4.0.2'
+gem 'rails', '4.0.3'
 
 gem 'pg'
 
 gem 'mail_view'
 gem 'decent_exposure'
 
+gem 'trello', git: 'git@github.com:dmitrytrager/trello.git'
```
Comment (VladimirMikhailov) [inline Gemfile:12]: `github: fs/trello` ?

**Comment by VladimirMikhailov** [comment]
+

File: Gemfile (hunk 1 above)
Comment (dmitrytrager) [inline Gemfile:12]: I was going to do it later, if this gem proves that it's ok)

Comment (VladimirMikhailov) [inline Gemfile:12]: ok, works for me. thanks =)

**Pull request merged**